src/NavigationActions.js

```diff
@@ -11,10 +11,15 @@ const RESET = 'Navigation/RESET';
 const SET_PARAMS = 'Navigation/SET_PARAMS';
 const URI = 'Navigation/URI';
 
-const createAction = (type: string) => (payload: Object = {}) => ({
-  type,
-  ...payload,
-});
+const createAction = (type: string) => {
+  const fn = (payload: Object = {}) => ({
+    type,
+    ...payload,
+  });
+
+  (fn: any).toString = () => type;
```

Comment:
Nit: Change the Flow type of `fn` to `mixed`. See https://flow.org/en/docs/types/functions/#toc-function-type

Reply:
I think the given explanation justifies the usage of `any`, so I'll consider this resolved.